### PR TITLE
[ADD] websites: add plausible.io Dashboard documentation

### PR DESCRIPTION
**Pull request by JKE-be**

### CODE REVIEW

**Comment by JKE-be** [comment]
_home_odoo_src_odoo_documentation__build_html_applications_websites_website_optimize_plausible html

File: content/applications/websites/website/optimize/plausible.rst [hunk 1]
```diff
@@ -0,0 +1,42 @@
+=======================================================
+How to track your website's traffic with Plausible.io
+=======================================================
```
Comment (Lau-Odoo) [inline content/applications/websites/website/optimize/plausible.rst:3]:
```suggestion
=====================================================
How to track your website's traffic with Plausible.io
=====================================================
```

File: content/applications/websites/website/optimize/plausible.rst [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+To follow your website's traffic with Plausible.io:
+
+- `Create a Plausible.io account <https://plausible.io/register/>`__ if   you don't have any.
```
Comment (Lau-Odoo) [inline content/applications/websites/website/optimize/plausible.rst:7]:
```suggestion
- `Create a Plausible.io account <https://plausible.io/register/>`__ if you don't have any.
```

File: content/applications/websites/website/optimize/plausible.rst [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+- `Create your website <https://plausible.io/sites/new/>`__ and `Start collecting data`  if it is not already done.
```
Comment (Lau-Odoo) [inline content/applications/websites/website/optimize/plausible.rst:9]:
```suggestion
- `Create your website <https://plausible.io/sites/new/>`__ and `Start collecting data` if it is not already done.
```

File: content/applications/websites/website/optimize/plausible.rst [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+- Go through the `list of yours websites <https://plausible.io/sites/>`__ and click on the gear wheel.
+
+    .. image:: media/plausible_cog.png
+        :align: center
```
Comment (Lau-Odoo) [inline content/applications/websites/website/optimize/plausible.rst:14]:
```suggestion
    .. image:: media/plausible_cog.png
       :align: center
```

File: content/applications/websites/website/optimize/plausible.rst [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+- Click on the button `+ New Link` in the tab Visibility to create your Shared links.
+
+    .. image:: media/plausible_new_sharedlink.png
+        :align: center
```
Comment (Lau-Odoo) [inline content/applications/websites/website/optimize/plausible.rst:19]:
```suggestion
    .. image:: media/plausible_new_sharedlink.png
       :align: center
```

File: content/applications/websites/website/optimize/plausible.rst [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+- Choose a name (and NO password - it is not supported to embed it in Odoo) and click on the button `Create Shared Link`
+
+    .. image:: media/plausible_create_sharedlink.png
+        :align: center
```
Comment (Lau-Odoo) [inline content/applications/websites/website/optimize/plausible.rst:24]:
```suggestion
    .. image:: media/plausible_create_sharedlink.png
       :align: center
```

File: content/applications/websites/website/optimize/plausible.rst [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+- Copy the new Shared Link url from Plausible.io and paste it in the Settings of Odoo
+
+    .. image:: media/plausible_copy_sharedlink.png
+        :align: center
```
Comment (Lau-Odoo) [inline content/applications/websites/website/optimize/plausible.rst:29]:
```suggestion
    .. image:: media/plausible_copy_sharedlink.png
       :align: center
```

File: content/applications/websites/website/optimize/plausible.rst [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+    .. image:: media/plausible_paste_sharedlink.png
+        :align: center
```
Comment (Lau-Odoo) [inline content/applications/websites/website/optimize/plausible.rst:31]:
```suggestion
    .. image:: media/plausible_paste_sharedlink.png
       :align: center
```

**Review by Lau-Odoo — APPROVED** [review]

File: content/applications/websites/website/optimize/plausible.rst [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+How to track your website's traffic with Plausible.io
+=====================================================
+
+To follow your website's traffic with Plausible.io:
+
+- `Create a Plausible.io account <https://plausible.io/register/>`__ if you don't have any.
+
+- `Create your website <https://plausible.io/sites/new/>`__ and `Start collecting data` if it is not already done.
+
+- Go through the `list of yours websites <https://plausible.io/sites/>`__ and click on the gear wheel.
```
Comment (rdeodoo) [inline content/applications/websites/website/optimize/plausible.rst:11]:
```suggestion
- Go through the `list of your websites <https://plausible.io/sites/>`__ and click on the gear icon.
```

File: content/applications/websites/website/optimize/plausible.rst [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+       :align: center
+
+
+.. note:: You are on the Odoo Saas Platform ?
+
+    When you `create your database on www.odoo.com/trial <https://www.odoo.com/trial>`__, we create it for you on our own server.
+    You don't have anything to do.
+
+
+.. note:: What's about the Goal ?
```
Comment (rdeodoo) [inline content/applications/websites/website/optimize/plausible.rst:40]:
```suggestion
.. note:: What about the Goal?
```
(and maybe just `What about Plausible Goal(s)`?)

File: content/applications/websites/website/optimize/plausible.rst [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+
+- Copy the new Shared Link url from Plausible.io and paste it in the Settings of Odoo
+
+    .. image:: media/plausible_copy_sharedlink.png
+       :align: center
+    .. image:: media/plausible_paste_sharedlink.png
+       :align: center
+
+
+.. note:: You are on the Odoo Saas Platform ?
```
Comment (rdeodoo) [inline content/applications/websites/website/optimize/plausible.rst:34]:
```suggestion
.. note:: You are on the Odoo Saas Platform?
```

**Review by rdeodoo — CHANGES_REQUESTED** [review]
Damn, cool!
Thanks for writting it

**Review by rdeodoo — APPROVED** [review]
@robodoo r+

**Comment by JKE-be** [comment]
@robodoo r+

it has been approved by 2 reviewers already

**Pull request closed without merging**